mscore/lyricseditor.cpp

```diff
@@ -0,0 +1,59 @@
+//=============================================================================
+//  LyricsEditor
+//
+//  Copyright (C) 2015-2020 Peter Jonas
```

Comment:
What if someone else modifies the file, does that header become 
`Copyright (C) 2015-2020 Peter Jonas and MuseScore BVBA`
?

Reply:
Yes, if that person is an employee of MuseScore BVBA, otherwise it would be more accurate for the individual to write their own name. However, there is no requirement to add your name to the copyright header as long as *somebody's* name is there. If you don't add your name then the convention in open source is that [edits are covered by the existing license](https://help.github.com/en/github/site-policy/github-terms-of-service#6-contributions-under-repository-license) (`inbound=outbound`). In other words, if you don't add your name then everyone pretends you did anyway.

Reply:
Good to know. Thank you.